### PR TITLE
Add GovGen chain

### PR DESCRIPTION
**Pull request by tbruyelle**

### CODE REVIEW

File: govgen/chain.json [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+	"staking": {
+		"staking_tokens": [
+			{
+				"denom": "ugovgen"
+			}
+		]
+	},
+	"codebase": {
+		"git_repo": "https://github.com/atomone-hub/govgen",
+		"recommended_version": "v1.0.1",
```
Comment (JeremyParish69) [inline govgen/chain.json:36]: Please copy this version info into the version array.

File: govgen/chain.json [hunk 2]
```diff
@@ -0,0 +1,226 @@
+{
+	"$schema": "../chain.schema.json",
+	"chain_name": "govgen",
+	"chain_id": "govgen-1",
+	"website": "https://govgen.io/",
+	"pretty_name": "GovGen",
+	"status": "live",
+	"network_type": "mainnet",
+	"bech32_prefix": "govgen",
```
Comment (JeremyParish69) [inline govgen/chain.json:9]: Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (tbruyelle) [reply govgen/chain.json:9]: done here https://github.com/satoshilabs/slips/pull/1687

**Comment by tbruyelle** [comment]
> Can you please changed the 8-space tabs tabs with just 2 spaces?

Done here [ed8d777](https://github.com/cosmos/chain-registry/pull/4000/commits/ed8d777153632bebc02c4a865b8a33c99b2b50d3)

Thanks for the review!

**Comment by zdeadex** [comment]
@JeremyParish69 all good for it, few wallets are using this repo as reference to add for custom chains support

**Comment by 3eph1r0th** [comment]
@tbruyelle Hi. Added some fixes for Allnodes nodes and submitted PR to your github account.

**Comment by tbruyelle** [comment]
> @tbruyelle Hi. Added some fixes for Allnodes nodes and submitted PR to your github account.

Thank you, merged!

**Pull request merged**